__tests__/app.test.js

```diff
@@ -217,6 +217,73 @@ describe("9. GET /api/articles/:article_id/comments", () => {
   });
 });
 
+describe("10. POST /api/articles/:article_id/comments", () => {
+  test("status 201: responds with posted comment", () => {
+    const newComment = {
+      username: "icellusedkars",
+      body: "Awesome post!",
+    };
+    const article_id = 3;
+    return request(app)
+      .post(`/api/articles/${article_id}/comments`)
+      .expect(201)
+      .send(newComment)
+      .then(({ body: { comment } }) => {
+        expect(comment).toBeInstanceOf(Object);
+        expect(comment).toHaveProperty("comment_id");
```

Comment:
I'd want to check the comment_id is what we expect it to be, so more specific check here